boards/shields/nrf700x_nrf54h20dk/Kconfig.shield

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+DT_COMPAT_NORDIC_NRF700X_SPI := nordic,nrf700x-spi
+
+config SHIELD_NRF700X_NRF54H20DK
+	def_bool $(shields_list_contains,nrf700x_nrf54h20dk)
```

Comment:
is this shield really named like this?

Reply:
Yes, this was the conclusion we had earlier https://github.com/nrfconnect/sdk-ic-next/pull/480#issuecomment-1710010189 we can revisit this.